cryptographic_estimators/MQEstimator/MQAlgorithms/kpg.py

```diff
@@ -68,41 +68,51 @@ def __init__(self, problem: MQProblem, **kwargs):
         self._n_reduced = n
         self._m_reduced = m
 
-    def time_complexity(self):
+    def _compute_time_complexity(self, parameters: dict):
         """
-        Return the time complexity of kpg algorithm
+        Return the time complexity of the algorithm for a given set of parameters
 
-        EXAMPLES::
+        TESTS::
```

Comment:
no tests in those functions. This should be tested via the estimator tests. this function should not make the impression to the user as he might be intended to call it